mingw-w64-git/PKGBUILD

```diff
@@ -104,17 +104,7 @@ pkgver() {
   printf "%s.%s" "${basever}" "$(git rev-parse --short HEAD)"
 }
 
-case "$CARCH" in
-i686)
-  SHAREDIR=/mingw32/share/git
-  ;;
-x86_64)
-  SHAREDIR=/mingw64/share/git
-  ;;
-aarch64)
-  SHAREDIR=/clangarm64/share/git
-  ;;
-esac
+SHAREDIR=$MINGW_PREFIX/share/git
```

Comment:
This is also some nice preparation for other upstream environments like UCRT64. While they would need additional changes in git-for-windows/git (or patches in msys2/MINGW-packages) it's a good first step.